lib/sqlbuilder/builder.go

```diff
@@ -268,10 +272,19 @@ func Map(item interface{}, options *MapOptions) ([]string, []interface{}, error)
 
 		for _, fi := range fieldMap {
 
+			// Check for deprecated tags and give suggestions on how to fix them.
+			deprecatedTags := map[string]string{
+				"stringarray": "postgresql.StringArray",
+				"int64array":  "postgresql.Int64Array",
+			}
+			for k, v := range deprecatedTags {
```

Comment:
edit: do we even need special tags here? lib/pq seems to be happily accepting []int64 for example https://github.com/lib/pq/blob/master/array.go#L29-L53